src/navigation/wallet/screens/KeySettings.tsx

```diff
@@ -214,24 +216,24 @@ const KeySettings = () => {
         if (syncWalletsLength) {
           message =
             syncWalletsLength === 1
-              ? 'New wallet found'
-              : `${syncWalletsLength} wallets found`;
+              ? t('New wallet found')
+              : syncWalletsLength + t(' wallets found');
```

Comment:
`${syncWalletsLength}` + t(' wallets found'); or Key interpolation `t('numOfWallets', {syncWalletsLength})` and update the translation.json